README.md

```diff
@@ -480,17 +484,7 @@ canDrink;
 // => undefined
 ```
 
-We specified `true` as the value to `switch` on. All of our `case`s are
-comparisons, and if the comparison returns `true` its statement(s) will be run.
-Because we forwent `break` statements, we allow the JavaScript engine to
-continue cascading down the `case`s **even if it matches one or more**. With
-`age` set to `20` in the above example, the first `case`, `age >= 21`, returns
-`false` and so the assignment of `canDrink` never happens. The engine then
-proceeds to the next `case`, `age >= 18`, which returns `true`, assigning the
-value `true` to `isAdult` and `canEnlist`. Since it encounters no `break`
-statement, the JavaScript engine then evaluates the final `case`, `age >= 16`,
-which also returns `true`. It assigns `true` to `canVote`, reaches the end of
-the `switch` statement, and exits.
+We specified `true` as the value to `switch` on. All of our `case`s are comparisons, and if the comparison returns `true` its statements will be run. Because we did not include `break` statements, we allow the JavaScript engine to continue cascading down the `case`s **even if it matches one or more**. With `age` set to `20` in the above example, the first `case`, `age >= 21`, returns `false` and so the assignment of `canDrink` never happens. The engine then proceeds to the next `case`, `age >= 18`, which returns `true`, assigning the value `true` to `isAdult` and `canEnlist`. Since it encounters no `break` statement, the JavaScript engine then evaluates the final `case`, `age >= 16`, which also returns `true`. It assigns `true` to `canVote`, reaches the end of the `switch` statement, and exits.
```

Comment:
From here on down, the new sections aren't word-wrapped at 80—small thing but I've been trying to catch it in my own lessons.